custom_components/tahoma/sensor.py

```diff
@@ -55,23 +57,39 @@ def unit_of_measurement(self):
         """Return the unit of measurement of this entity, if any."""
 
         if self.tahoma_device.uiclass == "TemperatureSensor":
-            return TEMP_CELSIUS  # TODO Retrieve core:MeasuredValueType to understand if it is Celsius or Kelvin
+            # TODO Retrieve core:MeasuredValueType to understand if it is Celsius or Kelvin
+            return TEMP_CELSIUS
 
         if self.tahoma_device.uiclass == "HumiditySensor":
             return UNIT_PERCENTAGE
 
         if self.tahoma_device.uiclass == "LightSensor":
             return "lx"
 
+        if self.tahoma_device.uiclass == "ElectricitySensor":
+            return "W"
```

Comment:
Is it W or Wh or kWh?

Reply:
if the former, we should use `homeassistant.const.POWER_WATT`, 
if one of the laters, we should use `homeassistant.const.ENERGY_WATT_HOUR` or `homeassistant.const.ENERGY_KILO_WATT_HOUR`